src/Resources/config/services/events.xml

```diff
@@ -26,5 +26,12 @@
             <argument type="service" id="sylius.command_bus" />
             <tag name="messenger.message_handler" bus="sylius.event_bus" />
         </service>
+
+        <service id="sylius_invoicing_plugin.event_listener.workflow.payment.produce_order_payment_paid"
+                class="Sylius\InvoicingPlugin\EventListener\Workflow\Payment\ProduceOrderPaymentPaidListener">
+            <argument type="service" id="sylius_invoicing_plugin.event_producer.order_payment_paid" />
+
+            <tag name="kernel.event_listener" event="workflow.sylius_payment.completed.complete" priority="100" />
+        </service>
```

Comment:
Move it to `events/workflow.xml` file instead and import it here.
Maybe also rename `events` to `listeners`?